test/verify/storagelib.py

```diff
@@ -445,7 +445,14 @@ def dialog_with_retry(self, trigger, values, expect):
         self.dialog_wait_close()
 
     def udisks_objects(self):
-        return json.loads(self.machine.execute("""python3 -c 'import dbus, json; print(json.dumps(dbus.SystemBus().call_blocking("org.freedesktop.UDisks2", "/org/freedesktop/UDisks2", "org.freedesktop.DBus.ObjectManager", "GetManagedObjects", "", [])))'"""))
+        return json.loads(self.machine.execute(
+            """python3 -c '
+import dbus, json
```

Comment:
Can you indent that?

Reply:
Does this work for you?
```py
    def udisks_objects(self):
        return json.loads(self.machine.execute(["python3", "-c", textwrap.dedent("""
            import dbus, json
            print(json.dumps(dbus.SystemBus().call_blocking(
                "org.freedesktop.UDisks2",
                "/org/freedesktop/UDisks2",
                "org.freedesktop.DBus.ObjectManager",
                "GetManagedObjects", "", [])))""")]))
```

We could alternatively also stop calling python completely, and use `busctl -j call` instead.

Reply:
Pro: This looks much nicer:
```py
    def udisks_objects(self):
        reply = self.machine.execute(["busctl", "call", "--json=pretty",
                                      "org.freedesktop.UDisks2", "/org/freedesktop/UDisks2",
                                      "org.freedesktop.DBus.ObjectManager", "GetManagedObjects"])
        return json.loads(reply)["data"][0]
```

Con: RHEL 8's systemd is too old to have a `--json` option :crying_cat_face: